sky/packages/sky/lib/src/widgets/input.dart

```diff
@@ -22,19 +22,22 @@ class Input extends StatefulComponent {
 
   Input({
     GlobalKey key,
+    String initialValue : '',
     this.placeholder,
     this.onChanged,
     this.keyboardType : KeyboardType_TEXT
-  }): super(key: key);
+  }): _value = initialValue, super(key: key);
 
   int keyboardType;
   String placeholder;
   StringValueChanged onChanged;
 
-  String _value = '';
+  String _value;
   EditableString _editableValue;
   KeyboardHandle _keyboardHandle = KeyboardHandle.unattached;
 
+  String get value => _value;
```

Comment:
Please remove this accessor.  No one out side this class should be reading this value.  Code that wants to be updated about new values should get an `onChanged` callback.